app/models/user.rb

```diff
@@ -28,6 +28,7 @@ class User < Common::RedisStore
   attribute :loa
   attribute :multifactor
   attribute :authn_context
+  attribute :mhv_icn
```

Comment:
Maybe a comment here about how this attribute differs from the icn in the MVI profile, and when it is set vs. not?